frontend/apps/marketing/src/components/section/Section.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
+
```

Comment:
Any reason why this is necessary?

Reply:
It was throwing that error in Contentful.

Reply:
I see the issue, I pushed a proposed suggestion to your PR here. I'd like to maximize our server-side rendering as much as possible, which means we should avoid using `use client` as much as possible. The proposed fix makes it so that only `ThemeProvider` is `use client` (rather than all of Section).

Reply:
Actually I just double checked, this PR doesn't seem to use `ThemeProvider` at all, the colors work without it using pure CSS. If that's the case, could we remove ThemeProvider? If we do need ThemeProvider, you can use this [alternative implementation](https://github.com/code-dot-org/code-dot-org/pull/64538/commits/f0b832deffaf1e930cc19d38b5f3fe87b869e9be)

Reply:
Just updated to remove `ThemeProvider` in this use case!